src/engine/Join.cpp

```diff
@@ -837,3 +837,11 @@ ad_utility::AddCombinedRowToIdTable Join::makeRowAdder(
       1, IdTable{getResultWidth(), allocator()}, cancellationHandle_,
       CHUNK_SIZE, std::move(callback)};
 }
+
+// _____________________________________________________________________________
+std::unique_ptr<Operation> Join::clone() const {
+  auto copy = std::make_unique<Join>(*this);
```

Comment:
As stated, I am not yet sure about thiis implicit pattern of "defaulted copy + then cloning the subtrees".